wordpress-mobile/WordPress-iOS
# Rich Text: Filter new lines.

This PR fixes an issue where new lines ( `\n` ) would cause doubled paragraph spacing when combined with P tags, and additional bullet points when combined with LI tags when converting an HTML formatted string to an NSAttributedString.

To test:
Wait until the refreshed comments are merged to develop.
The easiest practical test is to create a comment with a list and add an extra carriage return between the list items.  This is easiest to do with the web-editor in a O2/P2 theme.
Confirm the bug by viewing the issue in the refreshed comments UI.
Confirm the fix by switching to this branch and viewing the comment again. 

Confirm the updated test passes.

Needs review: @kurzee
